core/services/gateway/handlers/functions/orm.go

```diff
@@ -0,0 +1,130 @@
+package functions
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/functions/generated/functions_router"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+)
+
+//go:generate mockery --quiet --name ORM --output ./mocks/ --case=underscore
+
+type ORM interface {
+	FetchSubscriptions(offset, limit uint, qopts ...pg.QOpt) ([]CachedSubscription, error)
```

Comment:
Maybe let's call it GetSubscriptions()? "Fetch" might suggest fetching from the chain.